### PR TITLE
Add myself to roster

### PR DESCRIPTION
**Pull request by ebkalderon**
Thanks for the `git` talk. I'll continue to attend further MasonLUG and Patriot Hackers meetings!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,7 @@ Use this Format:
 | **Name** | **GitHub**| **Notes** |
 |:---------|:----------|:----------|
 |Lilas Dinh|[hatFolk](https://github.com/hatFolk)|Current President|
+|Eyal Kalderon|[ebkalderon](https://github.com/ebkalderon)|CEO & Cofounder of Buttermilk Nectar Swag Industries (BNIS), Inc.|
```
Comment (hatFolk) [inline README.md:7]: This is important

File: README.md [hunk 2]
```diff
@@ -4,6 +4,7 @@ Use this Format:
...
```
Comment (hatFolk) [inline README.md:9]: Yup

File: README.md [hunk 3]
```diff
@@ -4,6 +4,7 @@ Use this Format:
...
 
 
-[![Join the chat at https://gitter.im/hatFolk/masonlug-members](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/hatFolk/masonlug-members?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
\ No newline at end of file
+[![Join the chat at https://gitter.im/hatFolk/masonlug-members](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/hatFolk/masonlug-members?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```
Comment (hatFolk) [inline README.md:10]: Sure

**Comment by ebkalderon** [comment]
Closing due to extended inactivity. 🙃

**Pull request closed without merging**